demo/src/screens/componentScreens/ProgressBarScreen.tsx

```diff
@@ -51,37 +47,31 @@ export default class ProgressBarScreen extends Component {
           <Text $textDefault text70 style={styles.text}>
             Default
           </Text>
-          <ProgressBar
-            progress={progresses[0]}
-            style={styles.progressBar}
-          />
+          <ProgressBar progress={progresses[0]} style={styles.progressBar}/>
```

Comment:
Let's keep the PRs as small as possible, changes that are irrelevant to the specific change should be in a different PR or at least a different commit.